blog.php

```diff
@@ -1,21 +1,23 @@
 <?php include 'includes/header.php';?>
-    <h1 class="bottom-spacer">Blog</h1>
```

Comment:
You need to add the page title back in.

Reply:
Sorry, I have no idea why it's not there. I dont remember deleting it. It's done.